src/commands/prompts/start.ts

```diff
@@ -165,11 +174,24 @@ const askTimeSpent = async () => {
     };
 };
 
-export const askStartDetails = async (apiProvider: ApiProvider) => {
+const askTargetprocessEntityIfRequired = async (targetprocessApi: Targetprocess, id?: number) => {
+    if (id) {
+        return await getLoggableTargetprocessEntity(targetprocessApi, id);
+    }
+
+    return await askTargetprocessEntity(targetprocessApi);
+};
+
+export const askStartDetails = async (apiProvider: ApiProvider, tpId: number) => {
```

Comment:
Should this be `tpId?: number`?

Reply:
Yes, no idea why that compiled.. fixing now

Reply:
It compiled because you're always passing that number of parameters to that method even if the value was undefined.